### PR TITLE
Brooke Krajancich Member View

### PR DESCRIPTION
**Pull request by BKrajancich**
# *Member View for Brooke*

## :recycle: Current situation & Problem
Brooke does not have a member views section with a bio and photo

## :bulb: Proposed solution
Add a member views section for Brooke

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ ] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: CS342ApplicationUITests/InstructorsUITests.swift [hunk 1]
```diff
@@ -26,5 +26,8 @@ class InstructorsUITests: XCTestCase {
         
         app.collectionViews.buttons["Vishnu Ravi, MD"].tap()
         XCTAssert(app.staticTexts["Vishnu 👋"].waitForExistence(timeout: 0.5))
+        
+        app.collectionViews.buttons["Brooke Krajancich"].tap()
```
Comment (PSchmiedmayer) [inline CS342ApplicationUITests/InstructorsUITests.swift:35]: @BKrajancich I like that you have started submitting assignment 1 early; the view already looks great. I have seen that you currently experience a build error in the GitHub Actions that results from a failing test case. You can also reproduce this locally if you run the `InstructorUITests`. The problem is in this line:

The UI test navigates into the detail view of Vishnu and is then stuck here. You can simply navigate back out there by calling:
```suggestion
        app.navigationBars.firstMatch.buttons["CS342"].tap()
        
        app.collectionViews.buttons["Brooke Krajancich"].tap()
```
before your test call. 

I suggest you move your test into your separate `MemberViews` subfolder test and execute it there (then you don't have to navigate back out of Vishnu's view before that) or add the line I have stated above before you try to open your subview.

**Comment by BKrajancich** [comment]
@PSchmiedmayer I tried to resolve the merge conflicts, but I must have wrongly done one of the config files and now it won't open in Xcode. Any debugging suggestions?

**Comment by BKrajancich** [comment]
Decided to setup a clean branch. See #45
